CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
 
 ### Bug Fixes
 
+* Fixed a bug related to link queries (#4856).
```

Comment:
Fixed a bug in `isNull()`, `isNotNull()`, `isEmpty()` and `isNotEmpty()` when queries involve nullable fields in link queries.